README.md

```diff
@@ -63,20 +70,14 @@ pacman -S vivid
 
 Check out the [release page](https://github.com/sharkdp/vivid/releases) for binary builds.
 
-### Via cargo
-
-If you have Rust 1.31 or higher, you can install `vivid` from source via `cargo`:
-```
-cargo install vivid
-```
+## Theme Customization
 
-## Customization
+The bundled themes can be found in `$srcdir/themes` as a starting point.
 
-Copy the contents of the `share/vivid` folder in this repository to
-`/usr/share/vivid`, `$HOME/.config/vivid`, or `$XDG_CONFIG_PATH/vivid`
-on POSIX systems, or to `%APPDATA%\vivid` on Windows systems.
+Custom can be placed in `/usr/share/vivid`, `$HOME/.config/vivid`, or `$XDG_CONFIG_PATH/vivid`on POSIX systems,
```

Comment:
maybe: "Custom *filetype databases or themes* can be placed…"

Reply:
Whoops, accidentally deleted that!